src/test/fuzz/fuzz.cpp

```diff
@@ -56,6 +56,23 @@ static bool read_stdin(std::vector<uint8_t>& data)
 }
 #endif
 
+#if defined(PROVIDE_FUZZ_MAIN_FUNCTION)
+static bool read_file(fs::path p, std::vector<uint8_t>& data)
+{
+    uint8_t buffer[1024];
+    FILE *f = fsbridge::fopen(p.string(), "rb");
```

Comment:
Nit: Could `clang-format -i src/test/fuzz/fuzz.cpp` to get expected formatting? :)

Reply:
Thanks! Wasn't aware of `clang-format` yet!